homeassistant/components/netgear_lte/__init__.py

```diff
@@ -37,11 +37,18 @@
 EVENT_SMS = 'netgear_lte_sms'
 
 SERVICE_DELETE_SMS = 'delete_sms'
+SERVICE_SET_OPTION = 'set_option'
+SERVICE_CONNECT_LTE = 'connect_lte'
 
 ATTR_HOST = 'host'
 ATTR_SMS_ID = 'sms_id'
 ATTR_FROM = 'from'
 ATTR_MESSAGE = 'message'
+ATTR_FAILOVER = 'failover'
+ATTR_AUTOCONNECT = 'autoconnect'
```

Comment:
autoconnect - one or two words?

Reply:
I am not really an expert but I think it is correct in one word (possibly with a hyphen) while "auto connect" would mean to connect to a car or something like that :-)

https://dictionary.cambridge.org/dictionary/english/auto